DS and Algorithm/Day3/Selectionsort.py

```diff
@@ -0,0 +1,22 @@
+def selection_sort(arr):
+    n = len(arr)
+
+    for i in range(n):
+        # Find the minimum element in the unsorted part
+        min_index = i
+        for j in range(i-1, n):
```

Comment:
**Issue**: The inner loop starts from 'i-1' instead of 'i'. This will cause an index out of range error when 'i' is 0, and it will also lead to incorrect sorting behavior. <br> **Fix**: Change the inner loop to start from 'i' instead of 'i-1'. <br> **Code Suggestion**: 
 ```
 @@ -7,7 +7,7 @@ def selection_sort(arr):
      for j in range(i, n):
 ```